aymericbeaumet/metalsmith-clean-css
# feat(clean-css): Make clean-css a peer depedency

fixes #6 

... And a lot of other things completely unrelated.

Hi @aymericbeaumet, this PR contains:
- move to yarn workspaces. It allows requiring metalsmith-clean-css in
example easily and other quick wins like simpler travis setup
- add documentation on how to develop and release (quick interactive release script)
- upgrade all packages
- prepare renovate configuration, it will update all packages minor and
patch without you having to notic, to activate it: https://github.com/algolia/renovate-config-algolia#3-activate-renovate
- remove node4 testing
- simplify travis configuration, there should no more be needed that
many configs
- make clean-css a peer dependency
- change code to make it work with clean-css 4.1.9

Going further, to make this repo "modern hipster friendly", I would
switch testing framework to Jest and do a snapshot on the resulting
build folder of Metalsmith example.

Let me know what you think.
